frontend/src/components/UserProfile.tsx

```diff
@@ -18,6 +27,26 @@ export default function UserProfile() {
     user && user.date_joined
       ? new Date(user?.date_joined)
       : new Date(Date.now());
+  const newUserProperties = {
```

Comment:
These are just defaults, right? If so I’d recommend moving them outside of the component and caps casing the